reporters/SponsorReporter.js

```diff
@@ -33,15 +31,22 @@ class SponsorReporter {
       italic("- Each monitored wallet within the configuration object has their position and token balances printed")
     );
 
+    // Define the table rows beforehand to re-use the variables.
+    const row1Name = `Synthetic debt(${this.empProps.syntheticCurrencySymbol})`;
```

Comment:
Not the greatest names but I don't see an easy way to name them dynamically. You could possibly create an array of Strings and use that instead of naming each string